openedx/features/content_tagging/models.py

```diff
@@ -49,33 +49,42 @@ class Meta:
 
     @classmethod
     def get_relationships(
-        cls, taxonomy: Taxonomy, rel_type: RelType, org_short_name: str = None
+        cls,
+        taxonomy: Taxonomy,
+        rel_type: RelType,
+        org_short_name: Union[str, None] = None,
+        only_without_org: bool = True,
```

Comment:
@pomegranited We have two use cases for `org_short_name=None` here:

1. When we want to get all taxonomies that are NOT related to any org. We and to filter `org=None`.
2. When we DON'T want to apply an org filter.

Case 1 has a test in `test_api.py` (I don't know where we would use this)
Case 2 is what we need for the rest_api.

 I created the `only_without_org` parameter (and propagated it accordingly) here to differentiate. Let me know if it's ok or we should create another method for it. 
 
PS:  I'll try to come up with a better param name later. Suggestions are welcome.

Reply:
@rpenido 

> When we DON'T want to apply an org filter.

If you don't want to apply an org filter, you can use the plain `get_taxonomies` API method.

Reply:
:facepalm: You're right. My bad!